### PR TITLE
Saving user name

### PR DESCRIPTION
**Pull request by iisuslik43**

### CODE REVIEW

File: src/main/proto/cosmas.proto [hunk 1]
```diff
@@ -54,9 +54,10 @@ message FileVersion {
 
 message Patch {
     string userId = 1;
-    string text = 2;
-    int64 timestamp = 3;
-    string actualHash = 4;
+    string userName = 2;
```
Comment (dbarashev) [inline src/main/proto/cosmas.proto:57]: This is not the issue now, when this feature is not yet launched, but just for the record and for the future: these numbers are not just sequential numbers, they are embedded into the serialized protocol buffer instances and are used when deserializing to understand what byte goes where. Natually, if "2" used to be "text" and now becomes "userName" then we're having troubles. 
After launch we will not be able to afford changing the numbers (unless we appropriately change all the stored data).

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt [hunk 2]
```diff
@@ -543,6 +543,7 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
         return CosmasProto.Patch.newBuilder()
                 .setText(diffPatch)
                 .setUserId(COSMAS_ID)
+                .setUserName("COSMAS")
```
Comment (dbarashev) [inline src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt]: Let's use something like "Papeeria" just to have some understandable name if we happen to forget to update this later.

**Review by dbarashev — APPROVED** [review]
LGTM [1.5 comments]

File: src/main/proto/cosmas.proto (hunk 1 above)
Comment (iisuslik43) [reply src/main/proto/cosmas.proto:57]: ok, i ll remember it

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt (hunk 2 above)
Comment (iisuslik43) [reply src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt]: done

**Pull request merged**